src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class CreateBookRequestDto {
```

Comment:
since it is a DTO - we may use records :)

Reply:
It's a good idea, but unfortunately I can't use the records. Checkstyle failed with this feature